toc/README.md

```diff
@@ -0,0 +1,85 @@
+# TiDB Community Technical Oversight Committee (TOC)
+
+The TiDB Community TOC serves as the main bridge and channel for coordinating and information sharing across companies and organizations. It is the coordination center for solving problems in terms of resource mobilization, technical research and development direction in the current community and cooperative projects.
+
+## Responsibilities:
+
+- Partner cooperation project progress monitoring and guidance
+- Supervision and guidance on the progress of the projects in TiDB ecosystem.
+- Participate in technical route discussion and decision-making, such as roadmap formulation, etc.
+- Open source collaboration process discussion and decision
+- Discussion and decision-making of community-related activities
+
+## Membership
+
+TOC members include official members and non-voting members:
+
+- Official members have the right to participate in TOC meetings, propose proposals, and vote.
+- Non-voting members have the right to participate in TOC meetings, propose proposals, but do not have the right to vote.
+
+TOC members come from community partners who have signed a cooperation agreement with PingCAP and representatives of important community projects.
+
+## Nomination
+
+All official members of TOC have the right to nominate representative as new TOC member. The nominee is required to meet the following standards:
+
+- The nominee must have enough energy to invest in TOC affairs, such as participating in TOC discussions and [decision making](./proposal), and participating in TOC monthly meetings.
+- The nominee must have a deep understanding of TiDB or other projects in TiDB ecosystem.
+- The nominee must belong to a certain organization or project of TOC membership and be able to participate in TOC on behalf of his/her organization.
```

Comment:
or project?

Reply:
As our communication, there is no problem here.

Reply:
elaboration: organization can be company or project.